pkg/api/schemas/language.go

```diff
@@ -2,6 +2,7 @@ package schemas
 
 // UnifiedChatRequest defines Glide's Chat Request Schema unified across all language models
 type UnifiedChatRequest struct {
+	Model          string        `json:"model"`
```

Comment:
If we make the model field as required then we eliminate a possibility to send the simplest possible request which contains just chat history and the next message, right? I would keep that option to make Glide easier to use for people who don't want to leverage the override

Reply:
We don't have to require it, right? So if its not specified it's disregarded.

Reply:
@mkrueger12 right but how would that work if you omitted that for a few items? 🤔 
Like: `[nomodel, chat3.5, nomodel, nomodel]`?

Reply:
Omitted it in the API call?

Reply:
My basic thought is a request is sent to a router, the router selects a model, then we search the API request to see if that model was specified by the end user, if so we use that message. If no model was specified then we just pass the message as is.